sklearn/linear_model/least_angle.py

```diff
@@ -1087,6 +1087,7 @@ def __init__(self, fit_intercept=True, verbose=False, max_iter=500,
         self.max_n_alphas = max_n_alphas
         self.n_jobs = n_jobs
         self.eps = eps
+        self.fit_path = True
```

Comment:
I'm confused. @jnothman is this ok? I guess there's no harm and it doesn't seem to break anything?